src/main/java/com/dabsquared/gitlabjenkins/publisher/GitLabMessagePublisher.java

```diff
@@ -122,11 +122,11 @@ private String getResultIcon(Result result) {
         if (result == Result.SUCCESS) {
             return ":white_check_mark:";
         } else if (result == Result.ABORTED) {
-            return ":point_up:";
+            return "";
```

Comment:
I can only see a square on Chrome on Windows 8. Looks like this is not widely supported yet as it's part of Unicode 9.0 in 2016.